userenvs/stream8.json

```diff
@@ -5,11 +5,11 @@
     }
   },
   "userenv": {
-    "name": "stream",
-    "label": "CentOS Stream",
+    "name": "stream8",
+    "label": "CentOS Stream 8",
     "origin": {
-      "image": "registry.centos.org/centos/centos",
-      "tag": "8"
+      "image": "quay.io/centos/centos",
+      "tag": "stream8"
```

Comment:
Does a stream9 or "latest" exist?

Reply:
This seems "complicated".  There is a stream9 tag and a latest tag...but the latest tag hasn't been updated in quite awhile (1+ year) an even the stream8 tag is looking a bit dated at this point (3 months).  I figured that once we resolved the current problems that we would probably add a stream9 userenv, but I figured I shouldn't complicate the current fixes by doing that right now.

You can see the tags and their age here:

https://quay.io/repository/centos/centos?tab=tags